components/collective-navbar/index.js

```diff
@@ -258,7 +258,7 @@ const getDefaultCallsToActions = (collective, sections, isAdmin, isHostAdmin, Lo
     hasManageSubscriptions: isAdmin && get(features, 'RECURRING_CONTRIBUTIONS') === 'ACTIVE',
     hasDashboard: isAdmin && isFeatureAvailable(collective, 'HOST_DASHBOARD'),
     hasRequestGrant:
-      (isFund || get(settings, 'fundingRequest') === true) && expenseSubmissionAllowed(collective, LoggedInUser),
+      (isFund || get(settings, 'expenseTypes.hasGrant') === true) && expenseSubmissionAllowed(collective, LoggedInUser),
```

Comment:
Do we still want the `isFund ||`, since grants are now opt-in?

Reply:
Ops, you are right. We don't need that. I've corrected it. 👍🏽